### DISCUSSION TITLE
Turkish government issued certificate of hs enrollment not accepted as valid

### DISCUSSION THREAD

**Discussion by abulat189** [community/community#140138] [GitHub Education]
### Hi everyone,

I am a highschool student in Turkey. I do not have a student id or any other type of evidence since i am in a highschool. The best proof that is accepted all around my country is a certificate of being a student or "Öğrenci Belgesi" in Turkish. It is issued by the government and schools give it out for students to use as proof of studentship but github does not recognize it, as the government of Turkey is a "3rd party" what can i do?? 
Last year i used it and got verified no problem but this year im having problems. Thanks in advance!

**Comment by mamiiblt** [comment]
I'm in the exact same situation as you, I tried twice and both times it was rejected as third party.

**Comment by ghost** [comment]


**Reply by mamiiblt** [reply]
alakası yok dediklerinin ancak e devletten aldığım öğrenci belgesinin yarısını atınca (bilgilerin olduğu kısım) otomatik onayladı. Muhtemelen AI ile onaylama işlemlerini yürütüyorlar

![Screenshot_20241221-145108](https://github.com/user-attachments/assets/5de51bc1-5acb-4a0b-b5ff-379446fbc336)

arka kameradan şu şekil çekip atsanız (bulanık olmayacak) direkt onaylanır.

**Reply by mamiiblt** [reply]
ilk olarak yorum yaptığım zaman bende aynı sorundan müzdariptim, ardından çözdüm ve communityde de belirttim zaten. Polemiklik durum yok artık öğrenci belgeleri bile docx formatında düzenlenmiyor ve yetkililerden biri önceki konuların birinde Türkçe / İngilizce fark etmiyor demişti. Alakasızlık buradan doğuyor yani
